src/main/java/de/tum/in/www1/artemis/domain/exam/dto/converter/ExamDTOConverter.java

```diff
@@ -0,0 +1,17 @@
+package de.tum.in.www1.artemis.domain.exam.dto.converter;
+
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.stereotype.Component;
+
+import de.tum.in.www1.artemis.domain.exam.Exam;
+import de.tum.in.www1.artemis.domain.exam.dto.ExamDTO;
+
+@Component
+public class ExamDTOConverter implements Converter<Exam, ExamDTO> {
```

Comment:
we do not use such converters until now in the code. What would be the advantage?

Reply:
I think we do not really need the converters so I removed them. They were introduced to enable thread-safe conversion.

Nevertheless, we should define a way to convert entities to DTOs and vice versa. I think its not good practice to add the conversion methods to the entities and DTOs itself. We might come up with our own converter classes or with a library like [modelmapper](http://modelmapper.org/).

It would be nice to have something like the following: 
```
@PostMapping
public void createExam(@DTO(ExamInstructorDTO.class) Exam exam) {
     examRepository.save(exam);
}
```